client/my-sites/checkout/composite-checkout/test/composite-checkout.js

```diff
@@ -587,34 +585,16 @@ describe( 'CompositeCheckout', () => {
 		} );
 	} );
 
-	it( 'removes a product from the cart after clicking to remove it outside of edit mode', async () => {
-		const cartChanges = { products: [ planWithoutDomain, domainProduct ] };
-		render( <MyCheckout cartChanges={ cartChanges } />, container );
-		const activeSection = await screen.findByTestId( 'review-order-step--visible' );
-		const removeProductButton = await within( activeSection ).findByLabelText(
-			'Remove WordPress.com Personal from cart'
-		);
-		expect( screen.getAllByLabelText( 'WordPress.com Personal' ) ).toHaveLength( 2 );
-		fireEvent.click( removeProductButton );
-		const confirmModal = await screen.findByRole( 'dialog' );
-		const confirmButton = await within( confirmModal ).findByText( 'Continue' );
-		fireEvent.click( confirmButton );
-		await waitFor( async () => {
-			expect( screen.queryAllByLabelText( 'WordPress.com Personal' ) ).toHaveLength( 0 );
-		} );
-	} );
-
 	it( 'redirects to the plans page if the cart is empty after removing the last product', async () => {
 		const cartChanges = { products: [ planWithoutDomain ] };
 		render( <MyCheckout cartChanges={ cartChanges } />, container );
-		const editOrderButton = await screen.findByLabelText( 'Edit your order' );
-		fireEvent.click( editOrderButton );
 		const activeSection = await screen.findByTestId( 'review-order-step--visible' );
 		const removeProductButton = await within( activeSection ).findByLabelText(
 			'Remove WordPress.com Personal from cart'
 		);
 		fireEvent.click( removeProductButton );
-		const confirmButton = await screen.findByText( 'Continue' );
+		const confirmModal = await screen.findByRole( 'dialog' );
```

Comment:
There are now two `Continue` buttons when the dialog is displayed ( the second is from a further step which is now the active step since the order review cannot be ), so we need to be more specific about which we want.